src/wasm-stack.h

```diff
@@ -445,6 +445,8 @@ class BinaryenIRToBinaryWriter
   void emitFunctionEnd() {
     if (func->epilogLocation.size()) {
       parent.writeDebugLocation(*func->epilogLocation.begin());
+    } else {
+      parent.writeNoDebugLocation();
```

Comment:
Is this needed, since we prevent smearing across functions in another way anyhow? That is, this is the end of the function, so marking no-debug will not help any code but the next function, which is already handled by this PR elsewhere I think.

Reply:
This is to prevent the debug location on the last instruction to smear on the function epilogue (that is, the `end` opcode at the end of the function code). So that this:
```
(func
   ;;@ a:1:1
   (nop)
   ;;@
)
```
does not get turned into this:
```
(func
   ;;@ a:1:1
   (nop)
   ;;@ a:1:1
)
```

Reply:
I see, thanks. Maybe add a comment saying that?